package.json

```diff
@@ -20,7 +20,7 @@
     "prepublish": "npm run compile"
   },
   "peerDependencies": {
-    "node-sass": "^3.5.3"
+    "node-sass": "^4.0.0"
```

Comment:
Why `^4.0.0`? Since this package works with `node-sass` as well, I'd rather expect `>=3.5.3 <5.0.0`. `peerDependencies` shouldn't be more restrictive than needed.